templates/react/package.json

```diff
@@ -3,10 +3,9 @@
   "private": true,
   "scripts": {
     "build": "pnpm recursive run build",
-    "dev": "pnpm run initialize && pnpm recursive run dev",
+    "dev": "concurrently \"cd packages/contracts && pnpm run dev\" \"cd packages/client && pnpm run dev\"",
```

Comment:
this is to get around the `pnpm --filter` issues on windows right?

Reply:
also single quotes in these might be a little easier to read rather than escaped quotes?

Reply:
and `pnpm run recursive` doesn't like non-exiting dev scripts

Reply:
can use `pnpm recursive --no-bail`